### PR TITLE
Completed HTML Forms - Amelia Talkington

### PR DESCRIPTION
**Pull request by ameliatalkington**
Screen Shot 2021-01-19 at 12 56 23 PM

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <div>Email</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:9]: This should be a `label` element

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+        <input type="text" name="email" size="10" maxlenth="30"/>
+      <div>Why is coding cool?</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:11]: This should be a `label` element

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        <select name="Select an option">
+          <option value="why is coding cool">Because it's interesting</option>
+          <option value="why is coding cool">Because it's fun</option>
+          <option value="why is coding cool">Because it's powerful</option>
+        </select>
+      <div>Favorite Langauge (choose one)</div>
+      <div>
+        <input type="radio" name="code"/> HTML
```
Comment (codingcodymiller) [inline html-forms/index.html:19]: The text for these should be in a `label` element

File: html-forms/index.html [hunk 4]
```diff
@@ -0,0 +1,31 @@
...
+        <input type="radio" name="code"/> CSS
+        <input type="radio" name="code"/> Javascript
+      </div>
+      <div>Would you like more information?</div>
+        <input type="checkbox" name="signUp"/>Sign me up!
```
Comment (codingcodymiller) [inline html-forms/index.html:24]: Text should be in a `label` element 👍

File: html-forms/index.html [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+      <div>Comments</div>
```
Comment (codingcodymiller) [inline html-forms/index.html:25]: This should be a `label` element

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ameliatalkington** [comment]
Screen Shot 2021-01-19 at 1 53 39 PM

File: html-forms/index.html [hunk 6]
```diff
@@ -0,0 +1,51 @@
...
+<html>
+
+  <head>
+    <title>HTML Forms</title>
+  </head>
+
+  <body>
+    <h1>Coding Form!</h1>
+    <form action="">
+      <label for="email">Email
```
Comment (codingcodymiller) [inline html-forms/index.html:11]: The `for` attribute pairs with the `id` attribute on its inputs. Please add an `id` to each of your inputs to match the value of the `for` attribute on its label 👍

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by ameliatalkington** [comment]
Same output -- I might cry if it is still wrong

**Review by codingcodymiller — APPROVED** [review]
👍

**Pull request merged**